frontend/Auth/ResetPassword.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from "react";
+import { useHistory } from "react-router-dom";
+import { Button, ButtonKind } from "../Styleguide/Button";
+import { FormField } from "../Styleguide/FormField";
+import { FormFieldLabel } from "../Styleguide/FormFieldLabel";
+import { Input } from "../Styleguide/Input";
+import { RouterProps } from "react-router";
+import { useMutation } from "react-query";
+import { flaxFetch } from "../Utils/flaxFetch";
+import { unary } from "lodash-es";
+
+export function ResetPassword(props: RouterProps) {
+  const [resetPasswordFormData, setResetPasswordFormData] =
+    useState<ResetPasswordFormData>({
+      email: "",
+    });
+
+  const history = useHistory();
+
+  const submitMutation = useMutation<
+    ResetPasswordResultData,
+    Error,
+    React.FormEvent<HTMLFormElement>
+  >(
+    (evt) => {
+      evt.preventDefault();
+      let requestPromise = flaxFetch<ResetPasswordResultData>(
+        `/send-reset-password-email`,
+        {
+          method: "POST",
+          body: {
+            email: resetPasswordFormData.email,
+          },
+        }
+      );
+      return requestPromise;
+    },
+    {
+      onSuccess: async (data, variables, context) => {
+        history.push({
+          pathname: "/reset-password-email-sent",
+          state: { email: resetPasswordFormData.email },
+        });
+      },
+      onError: (error, variables, context) => {
+        console.log(error);
+      },
+    }
+  );
+
+  return (
+    <div className="flex justify-center h-screen">
+      <form onSubmit={unary(submitMutation.mutate)} className="pt-40 max-w-sm">
```

Comment:
Does pt-40 look good on mobile? Seems like a lot of padding

Reply:
Adjusted styling to remove pt-40.